src/Symfony/Bundle/FrameworkBundle/Resources/config/services.php

```diff
@@ -0,0 +1,201 @@
+use Symfony\Component\HttpKernel\UriSigner;
+use Symfony\Component\String\LazyString;
+use Symfony\Component\String\Slugger\AsciiSlugger;
+use Symfony\Component\String\Slugger\SluggerInterface;
+use Symfony\Component\Workflow\Event\AnnounceEvent;
+use Symfony\Component\Workflow\Event\CompletedEvent;
+use Symfony\Component\Workflow\Event\EnteredEvent;
+use Symfony\Component\Workflow\Event\EnterEvent;
+use Symfony\Component\Workflow\Event\GuardEvent;
+use Symfony\Component\Workflow\Event\LeaveEvent;
+use Symfony\Component\Workflow\Event\TransitionEvent;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+
+return static function (ContainerConfigurator $container) {
+    // this parameter is used at compile time in RegisterListenersPass
+    $container->parameters()->set('event_dispatcher.event_aliases', []);
+
+    $container->parameters()
+        ->set(ConsoleCommandEvent::class, 'console.command')
+        ->set(ConsoleErrorEvent::class, 'console.error')
+        ->set(ConsoleTerminateEvent::class, 'console.terminate')
+        ->set(PreSubmitEvent::class, 'form.pre_submit')
+        ->set(SubmitEvent::class, 'form.submit')
+        ->set(PostSubmitEvent::class, 'form.post_submit')
+        ->set(PreSetDataEvent::class, 'form.pre_set_data')
+        ->set(PostSetDataEvent::class, 'form.post_set_data')
+        ->set(ControllerArgumentsEvent::class, 'kernel.controller_arguments')
+        ->set(ControllerEvent::class, 'kernel.controller')
+        ->set(ResponseEvent::class, 'kernel.response')
+        ->set(FinishRequestEvent::class, 'kernel.finish_request')
+        ->set(RequestEvent::class, 'kernel.request')
+        ->set(ViewEvent::class, 'kernel.view')
+        ->set(ExceptionEvent::class, 'kernel.exception')
+        ->set(TerminateEvent::class, 'kernel.terminate')
+        ->set(GuardEvent::class, 'workflow.guard')
+        ->set(LeaveEvent::class, 'workflow.leave')
+        ->set(TransitionEvent::class, 'workflow.transition')
+        ->set(EnterEvent::class, 'workflow.enter')
+        ->set(EnteredEvent::class, 'workflow.entered')
+        ->set(CompletedEvent::class, 'workflow.completed')
+        ->set(AnnounceEvent::class, 'workflow.announce')
+    ;
+
+    $container->services()
+
+        // Autowireable parameters for the service definitions below
+        ->defaults()
+            ->bind('$debug', param('%kernel.debug%'))
+            ->bind('$deprecationLogsFilepath',
+                sprintf('%s/%sDeprecations.log', param('kernel.cache_dir'), param('kernel.container_class'))
+            )
+            ->bind('$defaultLocale', param('%kernel.default_locale%'))
+            ->autowire()
+            ->private()
+
+        ->set('parameter_bag', ContainerBag::class)
+            ->args([
+                service('service_container'),
+            ])
+            ->alias(ContainerBagInterface::class, 'parameter_bag')
+            ->alias(ParameterBagInterface::class, 'parameter_bag')
```

Comment:
please add a blank line between service definitions

Reply:
Fixed in bd44e756e985007c3a410ca149f07d4eef75517c